cmake/ipcl/ipcl-util.cmake

```diff
@@ -60,13 +60,13 @@ function(ipcl_detect_qat)
       execute_process(COMMAND lspci -d 8086:4940 COMMAND wc -l OUTPUT_VARIABLE QAT_PHYSICAL OUTPUT_STRIP_TRAILING_WHITESPACE)
       if(${QAT_PHYSICAL} GREATER_EQUAL "1")
         message(STATUS "Detected ${QAT_PHYSICAL} physical QAT processes")
+        ipcl_check_qat_service_status()
+        set(IPCL_FOUND_QAT TRUE PARENT_SCOPE)
         execute_process(COMMAND lspci -d 8086:4941 COMMAND wc -l OUTPUT_VARIABLE QAT_VIRTUAL OUTPUT_STRIP_TRAILING_WHITESPACE)
         if(${QAT_VIRTUAL} GREATER_EQUAL "1")
           message(STATUS "Detected ${QAT_VIRTUAL} virtual QAT processes")
-          ipcl_check_qat_service_status()
-          set(IPCL_FOUND_QAT TRUE PARENT_SCOPE)
         else()
-          message(WARNING "NO virtual QAT processors - IPCL_ENABLE_QAT set to OFF")
+	        message(STATUS "NO virtual QAT processors")
```

Comment:
@skmono That's correct. With this modification it will prevent to compile for QAT physical functions, for which it is supposed to work as well. Previously, it would only allow compilation for QAT if QAT devices were mapped to virtual functions.